content/en/guides/deployment/deployment-koyeb.md

```diff
@@ -0,0 +1,133 @@
+
+In your Nuxt application directory run the following command to install dependencies:
+
+```bash
+# Using yarn
+yarn
+
+# Using npm
+npm install
+```
+
+Once the dependencies are installed, launch your application and ensure everything is working fine:
+
+```bash
+# Using yarn
+
+yarn dev
+
+# Using npm
+
+npm run dev
+```
+
+## Dockerize your application
+
+To Dockerize your Nuxt application, you need to create a `Dockerfile` in your project directory containing the content below:
+
+```dockerfile
+FROM node:lts
+
+WORKDIR /app
+
+COPY . ./
+RUN yarn
+
+EXPOSE 3000
+
+ENV HOST=0.0.0.0
+ENV PORT=3000
+
+RUN yarn build
+
+CMD [ "yarn", "start" ]
+```
+
+If you prefer to use `npm` instead of `yarn` set the following content in your `Dockerfile`:
+
+```dockerfile
+FROM node:lts
+
+WORKDIR /app
+
+COPY . ./
+RUN npm install
+
+EXPOSE 3000
+
+ENV HOST=0.0.0.0
+ENV PORT=3000
+
+RUN npm run build
+
+CMD [ "npm", "run", "start" ]
+```
+
+To build the Docker image execute the following command:
+
+```bash
+docker build . -t <YOUR_DOCKER_HUB_USERNAME>/my-nuxt-project
+```
+
+This command will build the Docker image with the name <YOUR_DOCKER_HUB>/my-nuxt-project. Once the build is over, you can run a container using the image locally to validate everything is working as expected running:
+
+```bash
+docker run -p 3000:3000 <YOUR_DOCKER_HUB_USERNAME>/my-nuxt-project
+```
+
+Open your browser and navigate to http://localhost:3000 to view your project landing page.
+
+## Push your Docker image on the Docker hub
+
+Since our Docker image is built and functional in our test, we can now upload it to the Docker Hub. In your terminal run the command below to push the image:
+
+```bash
+docker push <YOUR_DOCKER_HUB_USERNAME>/my-nuxt-project
+```
+
+## Deploy the Nuxt application to production on Koyeb
+
+On the Koyeb Control Panel, click the **Create App** button.
+
+In the form, fill the `Docker image` field with the name of the image we previously created which should look like `<YOUR_DOCKER_HUB_USERNAME>/my-nuxt-project`.
+
+Check the box `Use a private registry` and, in the select field, click **Create Registry Secret**.
+
+A modal opens asking for:
+
+- a name for the Secret which will be created, we can use for instance `docker-hub-secret`
+- the registry provider to generate the secret containing your private registry credentials, in our case Docker Hub
+- your Docker Hub username and password. We recommend you to [generate an access token](https://hub.docker.com/settings/security) from the Docker Hub to use instead of your password.
+  Once you've filled all the fields, click the **Create** button.
+
+In the _Ports_ section, change the export port from `80` to `3000`, which is the port our Nuxt application is listening on. This setting is required to let Koyeb know which port your application is listening to and properly route incoming HTTP requests. We don't need to change the _Path_, our app will be available at the root of our domain: `/`.
```

Comment:
If the default port for Koyeb is `80` (and `PORT` environment variable is not available), we may include it in the docker image to reduce user steps for integrations. (also i _guess_ path can be somehow mentioned for [router.base](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-router/#base))

Reply:
Will fix the port to remove this step. About the `path` parameter the user has to set, it's a Koyeb related setting which indicates our global load balancer where to forward requests. i.e. if your router base path is set to`/apps` and  you set the Koyeb path parameter to `/`, the app will be accessible via `/apps`. if the Koyeb path is set to `/nuxt` the app will be accessible via `/nuxt/apps.